network-api/networkapi/wagtailpages/models.py

```diff
@@ -165,7 +168,7 @@ def get_context(self, request):
         Extend the context so that mini-site pages know what kind of tree
         they live in, and what some of their local aspects are:
         """
-        context = super(MiniSiteNameSpace, self).get_context(request)
+        context = super().get_context(request)
```

Comment:
I'm replacing python 2.7 "super" calls with the much more sensible python 3.x "super" in files that I'm touching for this PR.

Reply:
How were you able to remove the params for the new "super"?

Reply:
that's what they changed. Those parameters were a legacy python way of "knowing what it was actually supposed to find as own class" which made _no_ sense. Python 3 fixed that so that if there is a `self`, then python knows what its superclass is, and you don't need to specify this superfluous information anymore

Reply:
ahhhhh makes sense—cool! 😄